### DISCUSSION TITLE
cluster mode / multi-region deployment / read-replicas

### DISCUSSION THREAD

**Discussion by liesislukas** [meilisearch/product#575] [Feedback & Feature Proposal]
Looking for way to run meilisearch in multi-region setup on aws. Any hints? What i would expect from meilisearch is eventually consistent search solution. It can be single master, multi read nodes. In perfect world it would do multi-master, multi-region, like dynamodb global table. But it's a bit more challenging in perspective of sync, so single master & multiple read-nodes would be still perfect solution. meilisearch is not primary database, it's index for search, but it needs to be quick for 2 remote regions like N. America & Asia, at the moment it's hard. Any hints?

**Comment by liesislukas** [comment]
discussion at #215 started years ago but seems dead 💀 seems like it was in plans back in 2021 to be released but also seems it never happen

**Reply by gmourier** [reply]
Hello @liesislukas,

We just rebooted that subject on [this discussion](https://github.com/meilisearch/product/discussions/617) now that we have finally released Meilisearch v1.0

I'm sorry for the delay 🙇‍♂️

We have no ETA for that subject, but we hope to explore it more actively with the community's help anytime. First, we must define the first iteration scope by asking precise questions.

Feel free to jump into the discussion!
